pkg/assembler/graphql/schema/certifyVuln.graphql

```diff
@@ -99,6 +99,14 @@ type VulnerabilityMetaData {
   origin: String!
   "GUAC collector for the document"
   collector: String!
+  "cvss 2.0 score, defaults to empty string"
```

Comment:
This should be broken out into its own node that ties to the vulnerability. We should rename the `VulnerabilityMetaData` from `certifyVuln` to something like `scannerMetadata`. Move `VulnerabilityMetaData` into its own graphQL schema file.